packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -949,6 +975,12 @@ class SliverReorderableListState extends State<SliverReorderableList>
         continue;
       }
 
+      // Skip separators in separated lists - they should not be considered for drop positioning
```

Comment:
++ throughout this file. Imagine an engineer coming upon this code years from now. What is this talking about separators? :)